microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md

```diff
@@ -73,6 +73,23 @@ Issues are evaluated live while you edit your code.  As you type, you get feedba
 
 ![A possible issue being explained on an output element](microsoft-edge-devtools-extension-images/live-issue-reporting.png)
 
+#### Automated fixes and issue filtering
```

Comment:
```suggestion
#### Automated Quick Fixes
```
Is "issue filtering" covered in this h4 section?  What's the relation between Quick Fix UI & issue filtering?

Reply:
My PR clarifies this.